lib/http.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var req = require('request');
+var util = require('util');
+
+var HttpClient = require('soap').HttpClient;
+
+function LBHttpClient(options, connector) {
+  if (!(this instanceof LBHttpClient)) {
+    return new LBHttpClient(options, connector);
+  }
+  HttpClient.call(this, options);
+  this.connector = connector;
+}
+
+util.inherits(LBHttpClient, HttpClient);
+
+LBHttpClient.prototype.request = function(rurl, data, callback, exheaders, exoptions) {
+  var self = this;
+  var options = self.buildRequest(rurl, data, exheaders, exoptions);
+  var context = {
+    req: options
+  };
+
+  function work(context, done) {
```

Comment:
probably should start naming functions like this something less vague than `work`. How about `execute` since thats what the hook/operation is called?